Vic2ToHoI4/Source/V2World/Inventions.cpp

```diff
@@ -78,10 +83,11 @@ void Vic2::inventions::generateNums(const std::list<std::string>& inventionFiles
 
 void Vic2::inventions::processTechFile(const std::string& filename)
 {
-	registerKeyword(std::regex("[a-zA-Z0-9_\\.\\�\\�\\�\\�\\:\\&]+"), [this](const std::string& inventionName, std::istream& theStream){
-		inventionNumsToNames.insert(make_pair(inventionNumsToNames.size() + 1, inventionName));
-		commonItems::ignoreItem(inventionName, theStream);
-	});
+	registerKeyword(std::regex("[a-zA-Z0-9_\\.\\�\\�\\�\\�\\:\\&]+"),
```

Comment:
This regex is wrong. Just line up the characters, the backslashes can be removed entirely, as well as the dots, etc.